meson_options.txt

```diff
@@ -8,7 +8,8 @@ option('enable-uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('enable-uefi-labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('enable-dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('enable-amt', type : 'boolean', value : true, description : 'enable Intel AMT support')
-option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable Thunderbolt support')
+option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable legacy Thunderbolt support')
```

Comment:
I think it makes sense to have an option -- if you're building a distro for a Raspberry Pi you don't need the thunderbolt stuff. I also think the commit to remove the old plugin should be a new commit, not included in this one.